README.md

```diff
@@ -11,7 +11,7 @@ jumps that could confuse logs, servers, Tor, i2p, etc.
 
 This package contains the sdwdate time fetcher and daemon. No
 installation on remote servers required. To avoid conflicts, this daemon
-should be not be enabled together with ntp or tlsdated.
```

Comment:
Can you please update that in debian/control as well? (No matter if in the same or different pull request.)

README.md is actually generated from debian/control description.